c2corg_ui/templates/helpers/view.html

```diff
@@ -180,7 +180,11 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
            <app-card app-card-doc="doc"></app-card>
         % if showDelete:
         <app-delete-association child-doctype="${type}" parent-id="${document['document_id']}" child-id="doc.document_id"
-                                ng-if="userCtrl.auth.isAuthenticated()"></app-delete-association>
+                                ng-if="userCtrl.auth.isAuthenticated()
+                                % if document['doctype'] == 'routes':
+                                    && ${associations}.length > 1
+                                % endif
```

Comment:
``${associations}.length > 1`` doesn't look like a nice way to test how many documents are associated because ``${associations}`` will print the whole associations list in the HTML. Which could be huge.
Rather user a full python code: ``len(associations)``.

A more concise syntax would be
```
ng-if="userCtrl.auth.isAuthenticated() ${'&& ' + len(associations) + ' > 1' if document['doctype'] == 'routes' else ''}"
```

Mmmm, but that test will not be updated when the associations list is updated. But neither would the ``${associations}.length`` way I think!?